features.go

```diff
@@ -12,6 +12,8 @@ const (
 	TGFixERC721Feature = "tg:fix-erc721"
 	// Enables support for Binance contract mappings in the Binance Gateway contract
 	TGBinanceContractMappingFeature = "tg:binance-cm"
+	// Store Mainnet Gateway address in Gateway Go contract
+	TGGatewayVersion1_1 = "tg:v1.1"
```

Comment:
`TG` stands for Transfer Gateway, so this reads "Transfer Gateway Gateway Version 1.1", which is a bit redundant, better to call this `TGVersion1_1`